graphql/schemabuilder/function.go

```diff
@@ -111,6 +118,62 @@ func (sb *schemaBuilder) buildFederatedFunction(typ reflect.Type, m *method) (*g
 	return field, nil
 }
 
+// buildShadowObjectFederationFunction builds a federation object and a field func that takes the
+// federation keys as args and constructs the shadow object. This is used for federated subqueries.
+// {
+//    __federation {
+//     [ObjectName]-[Service] (keys: Keys) {
+//       subQuery
+// 		}
```

Comment:
Fix the intention?

Reply:
ping this.

Reply:
Hm, I fixed this is the last commit, maybe its a github UI thing? I tried reformatting again